core/_neat.scss

```diff
@@ -0,0 +1,10 @@
+// Neat 2.0.0.alpha.0
+// http://neat.bourbon.io
+// Copyright 2015 thoughtbot, inc.
+// MIT License
+
+@charset "UTF-8";
```

Comment:
tysongach added a note 11 days ago
@charset must be the first element, per https://developer.mozilla.org/en-US/docs/Web/CSS/@charset